src/os/bluestore/BlueStore.h

```diff
@@ -1741,6 +1741,7 @@ class BlueStore : public ObjectStore,
 	// sure those threads see waiters and signal qcond.
 	++kv_submitted_waiters;
 	if (_is_all_kv_submitted()) {
+	  --kv_submitted_waiters;
```

Comment:
nit: we can just do ++kv_submitted_waiters in else clause...

Reply:
But the comments: // set flag before the check because the condition
	// may become true outside qlock, and we need to make
	// sure those threads see waiters and signal qcond.
	++kv_submitted_waiters;
I think  move  in else condition is wrong.